app/src/main/res/xml/developer_preferences.xml

```diff
@@ -384,6 +384,10 @@
             android:key="@string/preference_key_show_search_tab_tooltip"
             android:title="@string/preference_key_show_search_tab_tooltip" />
 
+        <SwitchPreferenceCompat
+            android:key="@string/preference_key_customize_toolbar_tooltip"
+            android:title="@string/preference_key_customize_toolbar_tooltip" />
```

Comment:
Could you please move this tooltip under "Onboarding" category? (If possible, please also move the search tab tooltip as well).